src/pages/workspaces/workspace/Data.js

```diff
@@ -556,21 +556,28 @@ const WorkspaceData = _.flow(
                 ])],
                 () => div({ role: 'list', style: { fontSize: 14, lineHeight: '1.5' } }, [
                   _.map(([tableName, { count }]) => {
+                    const canCompute = !!(workspace?.canCompute)
                     return h(DataTypeButton, {
-                      buttonStyle: { borderBottom: 0, height: 40 },
+                      buttonStyle: canCompute ? { borderBottom: 0, height: 40 } : { borderBottom: 0, height: 40, color: colors.dark(0.25) },
+                      tooltip: canCompute ? (tableName ? `${tableName} (${count} row${count === 1 ? '' : 's'})` : undefined) :
```

Comment:
Would it read more easily if we newlined the first value since it has a lot going on?
```suggestion
                      tooltip: canCompute ?
                        tableName ? `${tableName} (${count} row${count === 1 ? '' : 's'})` : undefined :
```